packages/components/src/date-time/time.tsx

```diff
@@ -124,28 +120,30 @@ export function TimePicker( { is12Hour, currentTime, onChange } ) {
 	 * Function that sets the date state and calls the onChange with a new date.
 	 * The date is truncated at the minutes.
 	 *
-	 * @param {Object} newDate The date object.
+	 * @param {Moment} newDate The date object.
```

Comment:
I don't think there's a need for the type anymore in the JSDoc, since we're already adding the type via TS

```suggestion
	 * @param  newDate The date object.
```

This applies to other JSDocs comments in this PR (e.g `getMomentDate` in the `utils.ts` file)

Reply:
Heck yeah